config/locales/words_order.en.yml

```
@@ -8,4 +8,5 @@ en:
       select_words: 'Hold Shift to select the words between which you want to insert a hyphen.'
       select_words_step1: 'Choose the first word.'
       select_words_step2: 'Choose the second word.'
-      release_lexeme: 'Press Esc to deselect the token'
\ No newline at end of file
+      release_lexeme: 'Press Esc to deselect the token'
+      
```

тут тоже последняя строка должна быть пустой, совсем, без пробелов
```suggestion
      release_lexeme: 'Press Esc to deselect the token'

```

---

Исправлено